app/presenters/two_factor_authentication/selection_presenter.rb

```diff
@@ -69,8 +69,8 @@ def disabled?
 
     def login_label(type)
       case type
-      when 'backup_code'
-        t('two_factor_authentication.login_options.backup_code')
+      when 'personal_key'
```

Comment:
Are we changing `backup_code` to `personal_key` permanently?

Reply:
Not as such. I think `personal_key` gets removed from this switch statement on a different branch that's ahead of mine. Those label cases are moved to their respective set up/sign in selection presenter. I'll double check to make sure I'm not somehow adding that back in.
It should be gone from line ~72 as of this PR https://github.com/18F/identity-idp/pull/9528

Reply:
There may have been a merge conflict where the `personal_key` statement got added back accidentally. I'd expect it shouldn't be here.

Reply:
I have removed that from my branch.